targets/TARGET_NUVOTON/TARGET_M261/device/M261.h

```diff
@@ -0,0 +1,602 @@
+/**************************************************************************//**
+ * @file     M261.h
+ * @version  V1.0
+ * @brief    Peripheral Access Layer Header File
+ *
+ * @note
+ * Copyright (C) 2019 Nuvoton Technology Corp. All rights reserved.
+ *
+ ******************************************************************************/
+
+/**
+  \mainpage Introduction
+  *
+  *
+  * This user manual describes the usage of M261 device driver
+  *
+  * <b>Disclaimer</b>
+  *
+  * The Software is furnished "AS IS", without warranty as to performance or results, and
+  * the entire risk as to performance or results is assumed by YOU. Nuvoton disclaims all
+  * warranties, express, implied or otherwise, with regard to the Software, its use, or
+  * operation, including without limitation any and all warranties of merchantability, fitness
+  * for a particular purpose, and non-infringement of intellectual property rights.
+  *
+  * <b>Copyright Notice</b>
+  *
+  * Copyright (C) 2019 Nuvoton Technology Corp. All rights reserved.
+  */
```

Comment:
I don't think this license matches what we expect from contributions.
Please see: https://os.mbed.com/docs/mbed-os/v5.13/contributing/license.html

Do you have any other license available for this file?

Reply:
@SeppoTakalo , so far, seems many targets device folder still not follow your expectation of license, for ex: `mbed-os\targets\TARGET_STM\TARGET_STM32F4\TARGET_MTS_MDOT_F411RE\device` .
Anyway, I did re-license files of M261 device folder to fulfill SPDX-License-Identifier: Apache-2.0 in commit d863637441

Reply:
@SeppoTakalo, do you have any concern on this commit d863637 for re-license ?